doc/perf_tune.md

```diff
@@ -84,6 +89,78 @@ Please notice that this method is not a general solution since it may lead to se
 3. Run the AOT module by iwasm with `--disable-bounds-checks` option.
 
 > Note: The size of AOT file will be much smaller than the default, and some tricks are possible such as let the wasm application access the memory of host os directly.
-Please notice that if this option is enabled, the wasm spec test will fail since it requires the memory boundary check. For example, the runtime will crash when accessing the memory out of the boundary in some cases instead of throwing an exception as the spec requires.
+> Please notice that if this option is enabled, the wasm spec test will fail since it requires the memory boundary check. For example, the runtime will crash when accessing the memory out of the boundary in some cases instead of throwing an exception as the spec requires.
 
 You should only use this method for well tested wasm applications and make sure the memory access is safe.
+
+## 7. Use linux-perf
+
+Linux perf is a powerful tool to analyze the performance of a program, developer can use it to find the hot functions and optimize them. It is one profiler supported by WAMR. In order to use it, you need to add `--perf-profile` while running _iwasm_. By default, it is disabled.
+
+> [!CAUTION]
+> For now, only llvm-jit mode supports linux-perf.
+
+Here is a basic example, if there is a Wasm application _foo.wasm_, you'll execute.
+
+```
+$ perf record --output=perf.data.raw -- iwasm --perf-profile foo.wasm
+```
+
+This will create a _perf.data_ and a _jit-xxx.dump_ under _~/.debug.jit/_ folder. This extra file is WAMR generated at runtime, and it contains the mapping between the JIT code and the original Wasm function names.
+
+The next thing need to do is to merge _jit-xxx.dump_ file into the _perf.data_.
+
+```
+$ perf inject --jit --input=perf.data.raw --output=perf.data
+```
+
+This step will create a lot of _jitted-xxxx-N.so_ which are ELF images for all JIT functions created at runtime.
+
+> [!TIP]
+> add `-v` and check if there is output likes _write ELF image ..._. If yes, it means above merge is successful.
+
+Finally, you can use _perf report_ to analyze the performance.
+
+```
+$ perf report --input=perf.data
+```
+
+> [!CAUTION]
+> Using release build of llvm and iwasm will produce "[unknown]" functions in the call graph. It is not only because
+> of the missing debug information, but also because of removing frame pointers. To get the complete result, please
+> use debug build of llvm and iwasm.
+>
+> Wasm functions will not be affected.
+
+### 7.1 Flamegraph
+
+[Flamegraph](https://www.brendangregg.com/flamegraphs.html) is a powerful tool to visualize stack traces of profiled software so that the most frequent code-paths to be identified quickly and accurately. In order to use it, you need to record call graphs when running `perf record`
```

Comment:
`to be` or `can be`?
```suggestion
[Flamegraph](https://www.brendangregg.com/flamegraphs.html) is a powerful tool to visualize stack traces of profiled software so that the most frequent code-paths can be identified quickly and accurately. In order to use it, you need to record call graphs when running `perf record`
```